scripts/vue-cli-2/pr.md

```diff
@@ -10,10 +10,19 @@
    1. ${COMMAND_2}
    2. Add [cypress.json](cypress.json)
    3. Add [cypress/plugins/index.js](cypress/plugins/index.js)
-   4. Add [src/App.spec.ct.js](src/App.spec.ct.js), [src/components/HelloWorld.spec.ct.js](src/components/HelloWorld.spec.ct.js)
+   4. Add [src/App.spec.ct.js](src/App.spec.ct.js), [src/components/HelloWorld.spec.ct.js](src/components/HelloWorld.spec.ct.js), [src/Counter.vue](src/Counter.vue), [src/components/Counter.spec.ct.js](src/components/Counter.spec.ct.js)
```

Comment:
Can you split this up into 2 lines, eg.

```suggestion
   1. Update [src/App.vue](src/App.vue) to reference new [src/components/Counter.vue](src/components/Counter.vue) component
   1. Add [src/App.spec.ct.js](src/App.spec.ct.js), [src/components/HelloWorld.spec.ct.js](src/components/HelloWorld.spec.ct.js), [src/components/Counter.spec.ct.js](src/components/Counter.spec.ct.js) spec files
```

Also, FWIW using `1.` for every bullet point means you won't ever have to re-number them.